### ISSUE TITLE
 Unable to resolve module `AccessibilityInfo` from  "/node_modules/react-native-swipe-hidden-header"

### ISSUE THREAD

**Issue by devLaaziz** [fengliu222/react-native-swipe-hidden-header#2]
![simulator screen shot 3 juin 2017 14 17 00](https://cloud.githubusercontent.com/assets/12499618/26754307/e0848ce8-4867-11e7-9bda-69e48a4d5d76.png)

**Comment by peacechen** [comment]
`react` and `react-native` should not be in this library's dependencies.  It causes npm to install additional libs nested under its local node_modules which conflict with the parent project's node_modules.

The fix is to move `react` and `react-native` to `devDependencies`.  See PR #3

**Comment by damikdk** [comment]
I have same issue using yarn. Should I install this lib directly from this repo?

**Comment by peacechen** [comment]
The fix was merged but a new version hasn't been published to npm.  You'll need to point directly to this repo in your package.json.
